### DISCUSSION TITLE
Aspcore vs asp.netcore

### DISCUSSION THREAD

**Discussion by thunder-007** [TechEmpower/FrameworkBenchmarks#9218] [Q&A] [answered]
I see two bench marks here 3rd and 50th
https://www.techempower.com/benchmarks/#hw=ph&test=plaintext&section=data-r22
as per what I understand asp.netcore can handle only 4,675,156 requests per second 
![image](https://github.com/user-attachments/assets/f15a8c10-fd77-400b-bf51-8f5439e93aa7)
![image](https://github.com/user-attachments/assets/50e8f141-e9cf-48e0-80de-5079a1008cb4)
who is wrong here or is it just me not knowing the exact difference

**Comment by axtox** [comment, marked as answer]
If you hover on those two sections you'll see that:
1. Which is on 3rd says: "asp.net core [aspcore] (.NET; kestrel on Linux; None on Linux; Raw database connectivity, Platform class; Realistic approach) [Test 595]"
2. The one that on 50th - "ASP.NET Core [ben] (.NET; kestrel on Linux; Postgres on Linux; Micro ORM; Micro class; Realistic approach) [Test 1006]"

Meaning that these projects were written a little bit different, and perhaps by different authors, that's why the naming does not match.

[UPD]
I've tried to search the repo, here's my guess on where to locate those projects and see the difference by yourself:
1. The 3rd one [is here](https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/CSharp/aspnetcore/src/Minimal)
2. The 50th one [is here](https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/CSharp/ben)
